### PR TITLE
[quest] The missing diplomat pt14

### PR DESCRIPTION
**Pull request by PolarCookie**
Archmage Tervosh will teleport to Sentry Point, Dustwallow Marsh, if approaching with The Missing Diplomat pt14 (id:1265) in the log.
He'll give a little buff when you turn in this quest.

https://www.youtube.com/watch?v=6XeuwJnqSd4&feature=youtu.be&t=5m11s
http://www.wowhead.com/spell=7120/proudmoores-defense

**Target branch(es):** master

**Issues addressed:** Closes #

**Tests performed:** tested in-game

**Known issues and TODO list:**
need to make Tervosh do a casting animation.
Tervosh is supposed to actually teleport out of his tower, leaving it empty, but right now another Tervosh just spawns.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1494246194005123844.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1494246194005123844');
+
+DELETE FROM `areatrigger_involvedrelation` WHERE `id` in (302,1667);
+INSERT INTO `areatrigger_involvedrelation` VALUES (1667,1265);
+
+
+DELETE FROM `areatrigger_scripts` WHERE `id` = 302;
+INSERT INTO `areatrigger_scripts` VALUES (302,'at_sentry_point');
+
+UPDATE `creature_template` SET `ScriptName` = 'npc_archmage_tervosh' WHERE = 4967;
```
Comment (HeartWell) [inline data/sql/updates/pending_db_world/rev_1494246194005123844.sql]: Change:
UPDATE `creature_template` SET `ScriptName` = 'npc_archmage_tervosh' WHERE  `entry` = 4967;

File: data/sql/updates/pending_db_world/rev_1494246194005123844.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+DELETE FROM `creature_text` WHERE `entry` = 4967 AND `groupid` = 0 and `id` = 0;
+INSERT INTO `creature_text` VALUES (4967, 0, 0, 'Go with grace, and may the Lady\'s magic protect you.', 12, 0, 100, 0, 0, 0, 0, 0, 'SAY1');
```
Comment (HeartWell) [inline data/sql/updates/pending_db_world/rev_1494246194005123844.sql]: Add BroadCastText:
INSERT INTO `creature_text` VALUES (4967, 0, 0, 'Go with grace, and may the Lady\'s magic protect you.', 12, 0, 100, 0, 0, 0, 1751, 0, 'SAY1');

**Comment by Viste** [comment]
request done mb merge?

**Comment by Inifield** [comment]
Need to fix reaming TODO, if nobody does it, i'll try to do it

**Comment by PolarCookie** [comment]
@Inifield go for it.
To clarify, he doesn't do any cast emote when he teleports into the sentry tower.
About the second issue; if you manage to despawn the Theramore tower Tervosh for a minute, with a teleport animation as he leaves, that should be enough.

**Comment by Inifield** [comment]
@PolarCookie Hi could you do this https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/ thanks in advance

**Comment by PolarCookie** [comment]
http://www.wowhead.com/npc=4967/archmage-tervosh#comments:id=293291
Can't perma-remove him, he starts a quest for Maraudon.

This PR should be editable already, do you have write access to AC?

**Comment by Inifield** [comment]
Hi, yes I have write access, i'm trying to brainstorm an idea but so far nothing come to mind for the spell visual I couldn't find the original spell used but 799 works fine

**Comment by Inifield** [comment]
I will finish the entire questline (especially the end) this week if time allows me

**Pull request merged**